packages/ripple/README.md

```diff
@@ -64,6 +64,19 @@ const RippleIcon = withRipple(Icon);
 Wrap your Icon component with the HOC `withRipple`, which returns a component
 with a ripple capable surface.
 
+You also need to include the sass mixins in order for this to work.
```

Comment:
Move this to the "Styles" section (under Sass)

Reply:
And maybe add a note under the CSS section that says only hover/focus styles will show up